mypy/errors.py

```diff
@@ -300,7 +300,7 @@ def clear_errors_in_targets(self, path: str, targets: Set[str]) -> None:
             self.error_info_map[path] = new_errors
 
     def generate_unused_ignore_notes(self, file: str) -> None:
-        ignored_lines = self.ignored_lines[file]
+        ignored_lines = set(self.ignored_lines[file].values())
```

Comment:
Add a comment reminding that this is a set of actual `# type: ignore`s in the file.